caikit_tgis_backend/tgis_connection.py

```diff
@@ -221,13 +224,29 @@ def load_prompt_artifacts(self, prompt_id: str, *artifact_paths: List[str]):
             str,
             artifact_paths=artifact_paths,
         )
-        target_dir = os.path.join(self.prompt_dir, prompt_id)
+
+        target_dir = Path(self.prompt_dir) / prompt_id
```

Comment:
One day I'll actually learn `pathlib`!

Reply:
I really like the path building using `/` :)